broker.go

```diff
@@ -116,12 +116,26 @@ func (sb *crdbServiceBroker) Bind(
 	if _, err := plan.crdb.Exec(
 		fmt.Sprintf("GRANT ALL ON DATABASE %s TO %s", dbName, user),
 	); err != nil {
+		_, _ = plan.crdb.Exec(fmt.Sprintf("REVOKE ALL ON TABLE %s.* FROM %s", dbName, user))
```

Comment:
[nit] these could be put in a "cleanup := func() {}" to avoid repeating

Reply:
done